src/v3/test/integration/flow-okta-verify-enrollment.test.tsx

```diff
@@ -75,7 +74,12 @@ const createTestContext = async () => {
 };
 
 describe('flow-okta-verify-enrollment', () => {
+  let mockSystemTime: number;
+
   beforeEach(() => {
+    // Mock system time for triggering resend email reminder element
+    mockSystemTime = Date.now();
+    Date.now = jest.fn(() => mockSystemTime);
```

Comment:
See comment in authenticator-verification-email.test regarding `spyOn` for date.